### ISSUE TITLE
Search functionality

### ISSUE THREAD

**Issue by guysoft** [yuforium/activity-pub-docs#1]
I think one of the mode central features in groups is the ability to search. And that a group retains its memory.
I was wondering if you think it would make sense to touch on that.

Perhaps a way to request a search.

If a search returns all the history of the group or only up the point the user joined. Usually if you are part of a forum or group you can see the entire memory of that community.

**Comment by cpmoser** [comment]
@guysoft yes that is definitely one of the most important functions of a forum, since it's very topic focused it becomes a really helpful knowledge base.  That's probably a good thing to add to this repository - not necessarily one to add to the `context` document but maybe a new one should be created, I could see something like a `Topic` or subject matter resource being the coordinator of this.

**Comment by guysoft** [comment]
For now I will try and have a local search function.  Perhaps a federated one would make more sense later.
Started implementing, for now I managed to get profile and the basic webfinger stuff working
